### PR TITLE
add 100% coverage

### PR DESCRIPTION
**Pull request by ivoreis**

### CODE REVIEW

File: test/spec/iterable/intersectionSpec.js [hunk 1]
```diff
@@ -3,9 +3,15 @@
 import { intersection } from "../../../src/iterable/intersection";
 
 describe("intersection()", function () {
-    it("should yield the intersection of two sorted iterators", function () {
+    it("should yield the intersection of two sorted iterators when first element of 'this' is higher than the first element of the argument", function () {
```
Comment (jussi-kalliokoski) [inline test/spec/iterable/intersectionSpec.js]: do you mean "lower"?

File: test/spec/iterable/intersectionSpec.js [hunk 2]
```diff
@@ -3,9 +3,15 @@
...
         [...[1,2,3,4,5]::intersection([2,3,4,6], function (b) {
             return this - b;
         })].should.deep.equal([2,3,4]);
     });
+
+    it("should yield the intersection of two iterators when the first element of 'this' is lower than the first element of the argument", function () {
```
Comment (jussi-kalliokoski) [inline test/spec/iterable/intersectionSpec.js]: do you mean "higher"?

**Comment by jussi-kalliokoski** [comment]
Awesome, thank you! :heart: :+1:

File: test/spec/iterable/intersectionSpec.js (hunk 1 above)
Comment (ivoreis) [inline test/spec/iterable/intersectionSpec.js]: :) yes I do mean that. Sorry for that

Comment (jussi-kalliokoski) [inline test/spec/iterable/intersectionSpec.js]: No problem! ;)

**Comment by jussi-kalliokoski** [comment]
Great! Merged! :tada:

**Pull request merged**